### PR TITLE
Install RDO repo before Redis and Rabbit

### PR DESCRIPTION
**Pull request by paramite**
We need RDO repo to be able to install RabbitMQ and Redis.

### CODE REVIEW

File: playbook.yml [hunk 1]
```diff
@@ -19,11 +19,13 @@
 - name: Install Redis
   hosts: redis_hosts
   roles:
+    - rdo
     - redis
 
 - name: Install RabbitMQ
   hosts: rabbit_hosts
   roles:
+    - rdo
```
Comment (larsks) [inline playbook.yml:28]: Should this be a dependency of the rabbitmq role instead?  Or do you think it should be explicit?  (same question for redis, of course)

Comment (larsks) [reply playbook.yml:28]: Actually, never mind; see #65 for a different solution.

**Comment by paramite** [comment]
Closing in favor of #65

**Comment by larsks** [comment]
Actually closing :smile:

**Pull request closed without merging**